### PR TITLE
fix(route): 鱼塘热榜修复与优化

### PR DESCRIPTION
**Pull request by MyFaith**
## 完整路由地址 / Example for the proposed route(s)

```routes
/mofish/136
```
## 说明 / Note
- 修复鱼塘热榜时间错误
- 优化鱼塘热榜每日动图的显示方式

### CODE REVIEW

File: lib/routes/mofish/index.js [hunk 1]
```diff
@@ -18,12 +18,18 @@ module.exports = async (ctx) => {
...
-        item: data.map((item) => ({
-            title: item.Title,
-            pubDate: new Date(item.releaseTime).toUTCString(),
-            link: item.Url,
-            guid: item.id,
-        })),
+        item: data.map((item) => {
+            const isImage = Number(id) === 136;
+            const description = isImage ? `<p><img src="${item.Url}" referrerpolicy="no-referrer"><br></p>` : title;
+            const pubDate = new Date(item.CreateTime * 1000).toUTCString();
```
Comment (TonyRL) [inline lib/routes/mofish/index.js:24]: please use parseDate() utils
docs: https://docs.rsshub.app/joinus/pub-date.html

Comment (MyFaith) [reply lib/routes/mofish/index.js]: updated

File: lib/routes/mofish/index.js [hunk 2]
```diff
@@ -18,12 +19,17 @@ module.exports = async (ctx) => {
     ctx.state.data = {
         title,
         link: `https://mo.fish/`,
```
Comment (TonyRL) [inline lib/routes/mofish/index.js:21]:
```suggestion
        link: 'https://mo.fish/',
```

File: lib/routes/mofish/index.js [hunk 3]
```diff
@@ -18,12 +19,17 @@ module.exports = async (ctx) => {
...
-        description: title,
-        item: data.map((item) => ({
-            title: item.Title,
-            pubDate: new Date(item.releaseTime).toUTCString(),
-            link: item.Url,
-            guid: item.id,
-        })),
+        item: data.map((item) => {
+            const isImage = Number(id) === 136;
+            const description = isImage ? `<p><img src="${item.Url}" referrerpolicy="no-referrer"><br></p>` : title;
```
Comment (TonyRL) [inline lib/routes/mofish/index.js:24]: Please use art-template for HTML element. See https://github.com/DIYgod/RSSHub/discussions/8359
docs: https://docs.rsshub.app/joinus/script-standard.html#template

No need to add `referrerpolicy="no-referrer"`, RSSHub will add this automatically

**Comment by TonyRL** [comment]
Wrong image url

<details><summary>/mofish/136</summary>

```xml
<item>
<title>
<![CDATA[ 俄罗斯各大银行大幅上调消费贷款利率 ]]>
</title>
<description>
<![CDATA[ <p> <img src="https://www.jiemian.com/article/7155121.html" referrerpolicy="no-referrer"> <br> </p> ]]>
</description>
<pubDate>Tue, 01 Mar 2022 00:01:24 GMT</pubDate>
<guid isPermaLink="false">114334052</guid>
<link>https://www.jiemian.com/article/7155121.html</link>
</item>
```
</details>

**Pull request merged**